samples/saga/batching/sample.md

```diff
@@ -12,10 +12,10 @@ related:
 
 This sample shows how to control the task (work) generation via batching of the work items through Sagas. This pattern of throttled message generation can be helpful when the the rate of generation needs to be controlled:
 
-- When you need to generate lots of messages as part of one operation, the amount of messages can further delay other messages in the same queue.
+- When lots of messages are generated as part of an incoming message, the amount of messages can further delay other messages in the same queue.
```

Comment:
This isn't really true as the total number of workitems is identical. I think the essence here is that this allows for interleaved processing of very large batches of work items

Maybe it even helps if this is visualised:

Sequential batches due to batches generated massive amounts workitems at start
 
| A       | B       | C       |
|---------|---------|---------|
| 100.000 |         |         |
|         | 100.000 |         |
|         |         | 100.000 |

Paging workitems via saga:

| A      | B      | C      |
|--------|--------|--------|
| 25.000 |        |        |
|        | 25.000 |        |
|        |        | 25.000 |
| 25.000 |        |        |
|        | 25.000 |        |
|        |        | 25.000 |
| 25.000 |        |        |
|        | 25.000 |        |
|        |        | 25.000 |
| 25.000 |        |        |
|        | 25.000 |        |
|        |        | 25.000 |

Reply:
That's true. I think some visualization will help.

Reply:
@ramonsmits added a mermaid diagram, please see if it captures this.